frontend/src/views/camp/CampProgram.vue

```diff
@@ -11,8 +11,7 @@ Show all activity schedule entries of a single period.
         v-model="editMode"
         :shake="showReminder"
         :disabled-for-guest="!isContributor"
-        :message="$tc('views.camp.campProgram.guestsCannotEdit')"
```

Comment:
why did you remove the message here?

Reply:
This was the wrong message. It would always show the "guests cannot edit" message, even when one is a full collaborator. This was a mistake that slipped through in the previous refactoring in #3414. In that PR, we introduced a generic "guests cannot edit" message directly in the LockIcon component.
But I could remove this translation from our translation files.

Reply:
Removed the superfluous translations for now. If we want a customized message for guest write permissions on the picasso, we can always refactor the LockIcon component again.